### DISCUSSION TITLE
[V4] file_size validator can't custom error messages

### DISCUSSION THREAD

**Discussion by anggi117** [adonisjs/core#4111] [Help]
I have validator for file and put the file_size:1mb
the validator its work, but can't custom error message. can someone help me ?
![image](https://user-images.githubusercontent.com/42998655/227409268-e0fbd3a4-d3f3-46a9-8263-012bb120097a.png)
![image](https://user-images.githubusercontent.com/42998655/227409395-68c4d08e-5dfc-4fd2-8a95-40a0ab1bb874.png)

**Comment by thetutlage** [comment]
Because the message key is incorrect. Please refer the docs for the same https://docs.adonisjs.com/reference/validator/schema/file#custom-messages-options

**Reply by anggi117** [reply]
![image](https://user-images.githubusercontent.com/42998655/227893423-76c89016-00e8-4e90-9fc3-4d116bf6898f.png)
im already used that reference too, but still same. message can't change like customization
